### DISCUSSION TITLE
extract --exclude <catalog>

### DISCUSSION THREAD

**Discussion by nathanforce** [lingui/js-lingui#1015] [Ideas]
Thoughts on supporting an API like `extract --exclude <catalog>`? We have a need for "freezing" a given catalog at a point in time and would like to prevent future runs of the extract command from modifying the file.  `compile`/`loader` would continue to work as expected.

**Comment by nathanforce** [comment]
Another option would be to add something like `frozen: true` to `CatalogConfig`.

**Reply by semoal** [reply]
I like. a bit more this option, probably exlcude in the cli is complex to explain/understand what are you excluding, (files or catalogs?).

> frozen on the Catalogconfig should be enough :)

**Reply by nathanforce** [reply]
Maybe `status: 'active' | 'frozen'`? I don't want to be hasty and solve only my own needs here.

**Reply by semoal** [reply]
Well active could be just as default
